qiskit/tools/visualization/_text.py

```diff
@@ -401,13 +401,20 @@ def fillup_layer(names):  # pylint: disable=arguments-differ
 class TextDrawing():
     """ The text drawing"""
 
-    def __init__(self, json_circuit, reversebits=False, plotbarriers=True):
+    def __init__(self, json_circuit, reversebits=False, plotbarriers=True, line_length=None):
         self.json_circuit = json_circuit
         self.reversebits = reversebits
         self.plotbarriers = plotbarriers
+        self.line_length = line_length
         self.qubitorder = self._get_qubitorder()
         self.clbitorder = self._get_clbitorder()
 
+    def __str__(self):
```

Comment:
Would it make sense here to have the `__repr__` also `return self.single_string()`? I only ask because right now we've got to cast everything as a string.

Reply:
__repr__ is used by debuggers and I don't think you should have more than a single line there. The cast is necessary for the tests, but the use case is with `print`, right?

Reply:
It is, that makes sense to me